SectionedSurface-Aplitop-1/SectionedSurface-Aplitop-1.IFC

```diff
@@ -0,0 +1,87 @@
+#28=IFCROAD('1b7reoKwTFJxr7W5culxb$',$,'Road',$,'IfcRoad',$,$,$,.COMPLEX.,$);
+#29=IFCRELAGGREGATES('1nbAt7eYv4swrQ7kMUYrD3',#9,$,$,#25,(#28));
+#30=IFCALIGNMENT('3NYWW0yfD9BwdMM3yK9zLG',$,'Eje1',$,$,#24,$,$);
+#31=IFCALIGNMENTHORIZONTAL('1Lb11OMQ1AthVTi6V7jHiU',$,'Eje1',$,$,#24,#49,0.);
+#32=IFCRELNESTS('3Wkl6gqJj9$RKLoFB4xbkt',#9,$,$,#30,(#31,#51));
+#33=IFCALIGNMENTSEGMENT('0sVreI8Kn6tuw4UYDmeHtA',$,$,$,$,#24,$,#35);
+#34=IFCCARTESIANPOINT((286.47559897,239.79766567));
+#35=IFCALIGNMENTHORIZONTALSEGMENT($,$,#34,1.50346536986339,0.,0.,107.15812752,$,.LINE.);
+#36=IFCDIRECTION((0.999655739909883,0.0262374096515884));
+#37=IFCAXIS2PLACEMENT2D(#34,#36);
+#40=IFCCARTESIANPOINT((0.,0.));
+#41=IFCDIRECTION((1.,0.));
+#42=IFCVECTOR(#41,1.);
+#43=IFCLINE(#40,#42);
+#44=IFCCURVESEGMENT(.CONTINUOUS.,#37,IFCNONNEGATIVELENGTHMEASURE(0.),IFCNONNEGATIVELENGTHMEASURE(107.15812752),#43);
+#45=IFCRELNESTS('3U4tk8m5LAnhDZNYLR6Op1',#9,$,$,#31,(#33));
+#46=IFCCOMPOSITECURVE((#44),.F.);
+#47=IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Axis','Model',*,*,*,*,#19,$,.GRAPH_VIEW.,$);
+#48=IFCSHAPEREPRESENTATION(#47,'Axis','Curve2D',(#46));
+#49=IFCPRODUCTDEFINITIONSHAPE($,$,(#48));
+#50=IFCRELCONTAINEDINSPATIALSTRUCTURE('1I8l5qEL12MhbHe9PyUXYB',$,$,$,(#30),#25);
+#51=IFCALIGNMENTVERTICAL('2WjaJzeLn4Dh0c1wFGZNYU',#9,'Eje1',$,$,#24,#67);
+#52=IFCALIGNMENTSEGMENT('027HuZKDjAuvbEZ$TopRhG',$,$,$,$,#24,$,#53);
+#53=IFCALIGNMENTVERTICALSEGMENT($,$,0.,107.158,45.315043,0.0401646447302114,0.0401646447302114,$,.CONSTANTGRADIENT.);
+#54=IFCCARTESIANPOINT((0.,45.315043));
+#55=IFCDIRECTION((0.999194375250509,0.0401322870983622));
+#56=IFCAXIS2PLACEMENT2D(#54,#55);
+#59=IFCCARTESIANPOINT((0.,0.));
+#60=IFCDIRECTION((1.,0.));
+#61=IFCVECTOR(#60,1.);
+#62=IFCLINE(#59,#61);
+#63=IFCCURVESEGMENT(.CONTINUOUS.,#56,IFCNONNEGATIVELENGTHMEASURE(0.),IFCNONNEGATIVELENGTHMEASURE(107.244398741871),#62);
+#64=IFCRELNESTS('1Ekm3ZhxDAIxwnMiXhv18W',#9,$,$,#51,(#52));
+#65=IFCGRADIENTCURVE((#63),.F.,#46,$);
+#66=IFCSHAPEREPRESENTATION(#47,'Axis','Curve3D',(#65));
+#67=IFCPRODUCTDEFINITIONSHAPE($,$,(#66));
+#68=IFCFACILITYPART('2VJRykc916eO0F0TUHiUj9',$,$,$,$,#24,#87,$,.COMPLEX.,IFCROADPARTTYPEENUM(.ROADSEGMENT.),.LONGITUDINAL.);
+#69=IFCRELAGGREGATES('0nVIlPiJ1D0Opi4EVcPTpM',#9,$,$,#28,(#68));
+#70=IFCOPENCROSSPROFILEDEF(.CURVE.,'CrossProfile-1',.T.,(0.0310000000000001,3.5,3.5,0.056),(2.49643136367755,3.141592654,3.141592654,2.48087836828573),('119','30','0','30','119'));
+#72=IFCPOINTBYDISTANCEEXPRESSION(IFCLENGTHMEASURE(0.),3.531,0.,0.,#65);
+#73=IFCOPENCROSSPROFILEDEF(.CURVE.,'CrossProfile-2',.T.,(0.543,3.5,3.5,0.301),(2.47492598733333,3.141592654,3.141592654,2.47714082675747),('119','30','0','30','119'));
+#75=IFCPOINTBYDISTANCEEXPRESSION(IFCLENGTHMEASURE(30.),4.043,0.,0.,#65);
+#76=IFCOPENCROSSPROFILEDEF(.CURVE.,'CrossProfile-3',.T.,(0.663,3.5,3.5,0.,3.154),(2.47492598733333,3.141592654,3.141592654,3.141592654,3.80836500656817),('119','30','0','30','119','139'));
```

Comment:
My interpretation is according to this image:
![image](https://user-images.githubusercontent.com/25506404/122394428-42e00400-cf76-11eb-9c41-365f2e6bba97.png)
Is it possible that cross section at 60 should join 119 and 139 (at section 60) instead of 119 and 30, i.e. have the width 0 as last value?
Please also check that my interpretation is correct. Other than that, I think that the file LFTM!

Reply:
I have added a new IfcOpenCrossProfileDef at station 45. Now I think it looks better.
![image](https://user-images.githubusercontent.com/69001750/122417854-1504ba80-cf8a-11eb-8c10-462321007d49.png)

Reply:
Sorry, this is the right image

![image](https://user-images.githubusercontent.com/69001750/122417957-2e0d6b80-cf8a-11eb-96d7-8076eb728c6a.png)